functionary/requirements-dev.in

```diff
@@ -0,0 +1,8 @@
+# Constrain to versions in the requirements.txt
```

Comment:
All these requirements file changes are unrelated, but I realized the dependencies weren't included in a previous PR so I just fixed that here.